### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kirbyfern**
updated small changes like spelling and periods.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,5 +45,5 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/[USERN
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+The gem is available as an open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
```
Comment (hollomancer) [inline README.md]: @kboygit, 'open source' in this context is acting as an adjective, not a noun - this should not be changed.

Comment (kirbyfern) [reply README.md]: oh yea @hollomancer sorry

File: README.md [hunk 2]
```diff
@@ -43,5 +43,5 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/Operat
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+The gem is available as  open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
```
Comment (rickr) [inline README.md]: Looks like there's an extra space here.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Operationcode::Airtable
 
-This gem provides a common interface to airtables for the Operation Code platform.
+This gem provides a common interface to airtables for the Operation Code platform
```
Comment (rickr) [inline README.md]: I messed up this merge conflict a bit. Do you mind adding a `.` back here?

File: README.md (hunk 2 above)
Comment (kirbyfern) [reply README.md]: oops yes sorry

File: README.md (hunk 3 above)
Comment (kirbyfern) [reply README.md]: yes sir no prob

**Pull request merged**